pkgs/development/python-modules/azure-devops/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+, msrest
+}:
+
+buildPythonPackage rec {
+  version = "5.0.0b8";
```

Comment:
we typically don't include beta versions

Reply:
That's fair, this package isn't need by azure-cli, so I'm okay with skipping it until a stable release.